package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@bluedot-innovation/cordova-plugin",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I think we're making breaking API changes, yeah?

Reply:
hmmm, yeah.... 2.1.0?

Reply:
If we're making a breaking change it'll have to be 3.0.0